stdpopsim/cli.py

```diff
@@ -535,6 +537,36 @@ def time_or_model(arg, _arg_is_time=[True, ], parser=top_parser):
             help="Change to the specified simulation MODEL at generation T. "
                  "This option may provided multiple times.")
 
+    def slim_exec(path):
+        # Hack to set the SLIM environment variable at parse time,
+        # before get_version() can be called.
+        os.environ["SLIM"] = path
+        return path
+    slim_parser = top_parser.add_argument_group("SLiM specific parameters")
+    slim_parser.add_argument(
+            "--slim-path", metavar="PATH", type=slim_exec, default=None,
+            help="Full path to `slim' executable.")
+    slim_parser.add_argument(
+            "--slim-script", action="store_true", default=False,
+            help="Write script to stdout and exit without running SLiM.")
+    slim_parser.add_argument(
+            "--slim-scaling-factor", metavar="Q", default=10, type=float,
+            help="Rescale model parameters by Q to speed up simulation. "
+                 "See SLiM manual: `5.5 Rescaling population sizes to "
+                 "improve simulation performance`. "
+                 "[default=%(default)s].")
+    slim_parser.add_argument(
```

Comment:
is it worth having both this option and the "no-burn-in" option available at the CLI? I think it might make for a simpler UX if we ditch the latter option.

Reply:
I guess this ties in with having a parameter for the burn-in time. How about changing `--slim-no-burnin` to be `--slim-burnin X`, where X is the length of the burn-in period in units of N generations? The default could be 10, and the user can set it to 0 to disable burn in. And then this is still separated from whether the coalescence check is done.

In the future case of having background selection, probably we'll want to disable the coalescence check and fix a longer burn in time. (I tried some simulations with a distribution of fitness effects, which didn't complete coalesce before I killed the jobs after a couple of hours). Does this kind of use case warrant leaving in both options?